kmod/src/inode.c

```diff
@@ -265,6 +289,15 @@ static void load_inode(struct inode *inode, struct scoutfs_inode *cinode, int is
 	si->flags = le32_to_cpu(cinode->flags);
 	si->crtime.tv_sec = le64_to_cpu(cinode->crtime.sec);
 	si->crtime.tv_nsec = le32_to_cpu(cinode->crtime.nsec);
+	if (isize == SCOUTFS_INODE_FMT_V2_BYTES && !clear_bits) {
```

Comment:
This should be either setting the worm_ fields if the incoming inode item was large enough, or zeroing them.   It can key on the size and doesn't need clear_bits.   (and again maybe call it item_bytes or something instead of isize?)